client/src/components/Videos.js

```diff
@@ -0,0 +1,83 @@
+import { useState } from "react";
+import staticData from "../exampleresponse.json";
+import AddVideoForm from "./AddVideoForm";
+import Search from "./Search";
+import SingleVideo from "./SingleVideo";
+import "./Videos.css";
+
+const Videos = () => {
+  const sortedVideosData = staticData.sort((a, b) => b.rating - a.rating);
+  const [videosData, setVideosData] = useState(sortedVideosData);
+  const [searchValue, setSearchValue] = useState("");
+  const [showError, setShowError] = useState(false);
+
+  // delete video (for SingleVideo)
+  const handleDeleteVideoClick = (id) => {
+    const newVideosData = videosData.filter((video) => {
```

Comment:
Yes! A filter!

Don't forget that for arrow functions like this you can use an implicit return by writing:
`.filter(video => video.id !== id)`
It's called an implicit return because it removes the `return` and the curly braces